CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ option(LIB_PROTO_MUTATOR_EXAMPLES "Enable examples building" ON)
 option(LIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF
        "Automatically download working protobuf" OFF)
 option(LIB_PROTO_MUTATOR_WITH_ASAN "Enable address sanitizer" OFF)
+option(LIB_PROTO_MUTATOR_OSS_FUZZ "Target OSS-Fuzz (do not enable outside of OSS-Fuzz!)" OFF)
```

Comment:
LIB_PROTO_MUTATOR_EXAMPLES_USE_HEAD, OFF ?

Reply:
OR _MASTER

Reply:
@vitalybuka I can do "_LATEST" to avoid the problems of "_MASTER" and "_HEAD", sure. Pushed.